qiita_core/support_files/server.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
@squirrelo That is expected - you should go to the `Advanced` button and move forward. Note that the certificates that we are using here (even if you generate it in your machine) is not validated by any certificate authority, so chrome will complain.

Note that if we want this to go away, we have to require to our users to request a certificate to one of these authorities, which I don't think we want to do.

Also, we cannot add a certificate approved by any authority in the repository, otherwise anyone can use such certificate and appear safe, while in reality is stealing information.

I'm going to add documentation on how to create your certificate, but even if you create it using those steps, this warning is going to appear.